README.md

```diff
@@ -9,38 +9,220 @@ Each library has its own set of requirements. We recommend using a clean conda e
 1) Clone this repository.
 2) Install PyTorch by following the instructions here: https://pytorch.org/get-started/locally/
 3) Install the common requirements for all library by running `pip install -r requirements/requirements.txt`.
-4) If you wish to run NeMo, note that the benchmark currently needs CUDA 12.6 (`nvidia-smi` should output "CUDA Version: 12.6" or higher), to fix a problem in previous drivers for RNN-T inference with cooperative kernels inside of conditional nodes (see here: https://github.com/NVIDIA/NeMo/pull/9869)
-5) Install the requirements for each library you wish to evalaute by running `pip install -r requirements/requirements_<library_name>.txt`.
-6) Connect your Hugging Face account by running `huggingface-cli login`.
+4) Install the requirements for each library you wish to evaluate by running `pip install -r requirements/requirements_<library_name>.txt`.
+5) Connect your Hugging Face account by running `huggingface-cli login`.
+
+**Note:** If you wish to run NeMo, the benchmark currently needs CUDA 12.6 to fix a problem in previous drivers for RNN-T inference with cooperative kernels inside conditional nodes (see here: https://github.com/NVIDIA/NeMo/pull/9869). Running `nvidia-smi` should output "CUDA Version: 12.6" or higher.
```

Comment:
Just for my understanding, we also ran our Transformers benchmarks on the CUDA 12.6 right?

Reply:
Yep! Everything run on the same machine, same CUDA, same pytorch, same python, etc.